server/config/express.js

```diff
@@ -43,10 +43,16 @@ exports.configure = function configure(app) {
     resave: Boolean(process.env.SESS_RESAVE),
     saveUninitialized: Boolean(process.env.SESS_UNINITIALIZED),
     unset: process.env.SESS_UNSET,
-    cookie: { secure : true },
+    cookie: { httpOnly : true },
     retries: 50
   }));
 
+  // indicate that we are running behind a trust proxy and should use a secure cookie
```

Comment:
Nothing.  The application will not do any extra checks on the cookie for tampering or check HTTP headers.